src/components/Book.js

```diff
@@ -1,15 +1,21 @@
 /* eslint-disable react/prop-types */
+import axios from 'axios';
 import React from 'react';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/books';
 
 function Book({ title, author, id }) {
   const dispatch = useDispatch();
+  const apiUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/8bHEqSbfdyYUuS4cM1PJ/books/';
+  async function remove(id) {
+    dispatch(removeBook(id));
+    await axios.delete(`${apiUrl}${id}`);
+  }
```

Comment:
This should be an action creator within your Redux `books.js` file. Please fix it.

<img width="710" alt="Screen Shot 2022-08-25 at 7 43 34" src="https://user-images.githubusercontent.com/82426682/186667746-1540693b-c5be-43a6-83d1-fe732b614583.png">

Reply:
Thank you @vechicin for reviewing my project.